tests/integration/standard/test_rack_aware_policy.py

```diff
@@ -0,0 +1,48 @@
+import unittest
+
+from cassandra.cluster import Cluster, ExecutionProfile, EXEC_PROFILE_DEFAULT, NoHostAvailable
+from cassandra.policies import ConstantReconnectionPolicy, HostFilterPolicy, RackAwareRoundRobinPolicy, RoundRobinPolicy,  SimpleConvictionPolicy, \
+    WhiteListRoundRobinPolicy, ExponentialBackoffRetryPolicy
+from cassandra.pool import Host
+from cassandra.connection import DefaultEndPoint
+
+from tests.integration import PROTOCOL_VERSION, local, use_multidc, use_singledc, TestCluster
+
+from concurrent.futures import wait as wait_futures
+
+
+def setup_module():
+    use_multidc({'DC1': {'RC1': 2, 'RC2': 2}, 'DC2': {'RC1': 2}})
+
+class RackAwareRoundRobinPolicyTests(unittest.TestCase):
+    @classmethod
+    def setup_class(cls):
+        cls.cluster = Cluster(contact_points=["127.0.0.1", "127.0.0.2", "127.0.0.3", "127.0.0.4", "127.0.0.5", "127.0.0.6"], protocol_version=PROTOCOL_VERSION,
```

Comment:
Don't use hardcoded addresses 

Ccm is used here we can extract the addresses from ccm cluster, as we do in dtest (and probably in driver tests as well)


It's something like that (writing it from memory)
```python
[node.address() for node in ccm_cluster.nodelist()]
```